framework/wazuh/core/exception.py

```diff
@@ -361,8 +361,6 @@ class WazuhException(Exception):
         1800: {'message': 'Bad format in CDB list {path}'},
         1801: {'message': 'Wrong \'path\' parameter',
                'remediation': 'Please, provide a correct path'},
-        1802: {'message': 'Lists file not found',
-               'remediation': 'Please, use `GET /lists/files` to find all available lists'},
         1803: {'message': 'Error reading lists file',
                'remediation': 'Please, make sure you have read permissions over the file'
                },
```

Comment:
Errors 1803, 1804, 1806 (and likely 1805) correspond to lists and should therefore be removed as well